conf/messages

```diff
@@ -98,6 +98,40 @@ setup.footer.copyright=&copy; 2015 yetu AG
 setup.footer.privacy=Privacy Policy
 setup.footer.imprint=Imprint
 setup.footer.terms=Terms &amp; conditions
+setup.next=next &gt;
+
+#setup registration page
+setup.registration.register=Register at yetu
+setup.registration.already=Already registered?
+setup.registration.hint=all fields are mandatory
+setup.registration.firstname=First name
+setup.registration.lastname=Last name
+setup.registration.email=E-mail
```

Comment:
I will remove the old ones, when we change the process and replace old pages. So then they will not be there anymore.